tools/make/lib/install/modules.mk

```diff
@@ -45,10 +28,8 @@ $(npm_install_deps_file): $(ROOT_PACKAGE_JSON) $(print_npm_install_deps)
 # @example
 # make install-node-modules
 #/
-install-node-modules: $(npm_install_deps_file)
-	$(QUIET) $(MV) "$(ROOT_PACKAGE_JSON)" "$(ROOT_PACKAGE_JSON).copy"
-	$(QUIET) echo '{"name":"@stdlib/stdlib","version":"0.0.0"}' > "$(ROOT_PACKAGE_JSON)"
-	$(QUIET) { $(CAT) $(npm_install_deps_file) | xargs $(NPM) install --no-save && $(MV) -f "$(ROOT_PACKAGE_JSON).copy" "$(ROOT_PACKAGE_JSON)"; } || { $(MV) -f "$(ROOT_PACKAGE_JSON).copy" "$(ROOT_PACKAGE_JSON)" && exit 1; }
+install-node-modules: $(ROOT_PACKAGE_JSON)
+	$(QUIET) $(NPM) install --no-save
```

Comment:
This won't work for us. We don't actually want to install top-level namespaces when doing `git clone` for local development within a standalone repository. How the `make` recipe is configured is intentional, as it skips top-level namespace installation.